zipkin-autoconfigure/ui/src/test/java/zipkin/autoconfigure/ui/ITZipkinUiAutoConfiguration.java

```diff
@@ -32,7 +33,7 @@
 )
 public class ITZipkinUiAutoConfiguration {
 
-  @LocalServerPort int zipkinPort;
+  @Autowired @Value("${local.server.port}") int zipkinPort;
```

Comment:
I think it should just be `@Value("${local.server.port}")` without the `@Autowired`